storage/innobase/xtrabackup/src/xbcloud/http.cc

```diff
@@ -583,7 +585,7 @@ bool Http_client::make_async_request(const Http_request &request,
   using std::placeholders::_1;
   using std::placeholders::_2;
 
-  auto cb = std::bind(async_result_callback, callback, action, h, _1, _2);
+  auto cb = std::bind(async_result_callback, callback, h, _1, _2);
```

Comment:
Why have u removed this action string?

Reply:
@satya-bodapati I have made it more generic. Instead of passing action from xbcloud -> http_client -> make_async_request -> async_result_callback (which is where we used) We now just report that operation failed on object. 
Users cannot make a mix of operations on a single call to xbcloud, you will either put / get / delete.

Reply:
Failed to upload object vs Failed to download object, User will see like this now?

Reply:
They will see:

```
210625 08:25:06 /work/pxb/ins/2.4/bin/xbcloud: Operation failed. Error: Server returned nothing (no headers, no data)
```

Reply:
ok, a line before this there "Downloading.." or "Uploading file.." right?

Reply:
Not before, just when we complete.

```
210625 09:42:41 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/mysql/innodb_index_stats.ibd.00000000000000000001, size: 42
210625 09:42:42 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/mysql/time_zone_transition_type.ibd.00000000000000000000, size: 98373
210625 09:42:42 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/mysql/time_zone_transition_type.ibd.00000000000000000001, size: 49
210625 09:42:42 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sys/sys_config.ibd.00000000000000000000, size: 98356
210625 09:42:42 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sys/sys_config.ibd.00000000000000000001, size: 32
210625 09:42:42 /work/pxb/ins/2.4/bin/xbcloud: Operation failed. Error: Server returned nothing (no headers, no data)
210625 09:42:42 /work/pxb/ins/2.4/bin/xbcloud: Sleeping for 2384 ns before retrying marce6/sbtest/sbtest6.ibd.00000000000000000000 [1]
210625 09:42:45 /work/pxb/ins/2.4/bin/xbcloud: Operation failed. Error: Failed sending data to the peer
210625 09:42:45 /work/pxb/ins/2.4/bin/xbcloud: Sleeping for 2887 ns before retrying marce6/sbtest/sbtest6.ibd.00000000000000000001 [1]
210625 09:42:49 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sbtest/sbtest6.ibd.00000000000000000002, size: 10485812
210625 09:42:50 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sbtest/sbtest6.ibd.00000000000000000003, size: 10485812
210625 09:42:50 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sbtest/sbtest6.ibd.00000000000000000004, size: 10485812
210625 09:42:51 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sbtest/sbtest6.ibd.00000000000000000005, size: 10485812
210625 09:42:52 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sbtest/sbtest6.ibd.00000000000000000000, size: 10485812
210625 09:42:52 /work/pxb/ins/2.4/bin/xbcloud: successfully uploaded chunk: marce6/sbtest/sbtest6.ibd.00000000000000000001, size: 10485812
```

But the operation is always the same, user will be either uploading or downloading an object